Upstream-Status: Pending

Without this patch it fails with:
| make[6]: Entering directory `/OE/shr-core/tmp/work/armv7a-vfp-neon-oe-linux-gnueabi/abiword-2.5.2-r2/abiword-2.5.2/src/wp/ap/unix'
| make[6]: Nothing to be done for `install-exec-am'.
| test -z "/usr/share/abiword-2.5/glade" || /bin/mkdir -p "/OE/shr-core/tmp/work/armv7a-vfp-neon-oe-linux-gnueabi/abiword-2.5.2-r2/image/usr/share/abiword-2.5/glade"
|  /usr/bin/install -c -m 644 ap_UnixDialog_Break.glade ap_UnixDialog_Field.glade ap_UnixDialog_FormatTOC.glade ap_UnixDialog_FormatFootnotes.glade ap_UnixDialog_FormatTable.glade ap_UnixDialog_FormatFrame.gla
de ap_UnixDialog_Goto.glade ap_UnixDialog_InsertTable.glade ap_UnixDialog_Insert_DateTime.glade ap_UnixDialog_MailMerge.glade ap_UnixDialog_MetaData.glade ap_UnixDialog_Options.glade ap_UnixDialog_HdrFtr.glade
 ap_UnixDialog_Options_ColorSel.glade ap_UnixDialog_PageNumbers.glade ap_UnixDialog_PageSetup.glade ap_UnixDialog_Replace.glade ap_UnixDialog_New.glade ap_UnixDialog_Styles.glade ap_UnixDialog_Stylist.glade ap
 _UnixDialog_Stylist_modal.glade ap_UnixDialog_Tab.glade ap_UnixDialog_WordCount.glade ap_UnixDialog_Latex.glade ap_UnixDialog_Options.glade ap_UnixDialog_HdrFtr.glade '/OE/shr-core/tmp/work/armv7a-vfp-neon-oe-
 linux-gnueabi/abiword-2.5.2-r2/image/usr/share/abiword-2.5/glade'
 | /usr/bin/install: will not overwrite just-created `/OE/shr-core/tmp/work/armv7a-vfp-neon-oe-linux-gnueabi/abiword-2.5.2-r2/image/usr/share/abiword-2.5/glade/ap_UnixDialog_Options.glade' with `ap_UnixDialog_O
 ptions.glade'
 | /usr/bin/install: will not overwrite just-created `/OE/shr-core/tmp/work/armv7a-vfp-neon-oe-linux-gnueabi/abiword-2.5.2-r2/image/usr/share/abiword-2.5/glade/ap_UnixDialog_HdrFtr.glade' with `ap_UnixDialog_Hd
 rFtr.glade'
 | make[6]: *** [install-gladeDATA] Error 1
 | make[6]: Leaving directory `/OE/shr-core/tmp/work/armv7a-vfp-neon-oe-linux-gnueabi/abiword-2.5.2-r2/abiword-2.5.2/src/wp/ap/unix'

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>

diff -uNr abiword-2.5.2.orig/src/wp/ap/unix/GNUmakefile.am abiword-2.5.2/src/wp/ap/unix/GNUmakefile.am
--- abiword-2.5.2.orig/src/wp/ap/unix/GNUmakefile.am	2007-06-04 17:17:01.000000000 +0200
+++ abiword-2.5.2/src/wp/ap/unix/GNUmakefile.am	2011-10-18 00:48:05.247378444 +0200
@@ -69,9 +69,7 @@
 	ap_UnixDialog_Stylist_modal.glade	\
 	ap_UnixDialog_Tab.glade	\
 	ap_UnixDialog_WordCount.glade \
-	ap_UnixDialog_Latex.glade \
-	ap_UnixDialog_Options.glade \
-	ap_UnixDialog_HdrFtr.glade
+	ap_UnixDialog_Latex.glade
 
 if ENABLE_SPELL
 spellsrc=	ap_UnixDialog_Spell.cpp
